appeals/web/src/server/lib/mappers/data/initialise-and-map-data.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import('./types.js').InitialiseAndMapDataFactory} */
+export const initialiseAndMapDataFactory = (
```

Comment:
This name is a bit ambiguous especially if you're not very familiar with the codebase. I'd suggest maybe `getInitialiseAndMapDataFunction` or something similar